CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## dev
 
+- Fix FileNotFoundError when updating package cache on Windows (#1000)
```

Comment:
This doesn’t need a changelog since there’s no changes visible to end users.

Reply:
Hi @uranusjr! Thanks for the feedback, I have removed the changelog entry 👍